SU-UHR/stanford_bean_types
# Caw 816 icon grid

# READY FOR REVIEW

# Summary
- Cardinal at Work wants icon grid

# Needed By (Date)
- ASAP

# Urgency
- Moderate

# Steps to Test

1. Clone cardinal at work on anchorage to local
2. Use this branch for module
3. Enable stanford_icon_grid
4. clear cache
5. Make sure the icon grid block works as expected.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/CAW-816
- There will be an associated PR in https://code.stanford.edu/sws/cardinalatwork when there is a tagged version
